### PR TITLE
Document how to upgrade ddev, fixes #699

### LINKED ISSUE #699: document how to upgrade ddev
## What happened (or feature request):
It would be nice if there was a documentation how to upgrade ddev to newest version on each system.
I've figured it out that just calling installaton script works on linux, but it's not obvious.

### PR DESCRIPTION
**Pull request by tmotyl**
## The Problem/Issue/Bug:
Fixes https://github.com/drud/ddev/issues/699

### CODE REVIEW

**Comment by rfay** [comment]
Fantastic reminder. This confuses people lots of the time. The amazing thing is that because it's a golang executable, everything's bundled inside, and there's nothing but an executable copy. It's easy for us to forget because it's so easy.

File: docs/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ ddev requires ports 80 and 443 to be available for use on your system when proje
 
 If you need to use another environment after using ddev, just stop or remove all of your ddev projects. ddev only occupies system ports when at least one project is running.
 
-## Installation
+## Installation / upgrade
```
Comment (rfay) [inline docs/index.md]: "Installation/Upgrade"

File: docs/index.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ ddev requires ports 80 and 443 to be available for use on your system when proje
...
 ### Homebrew - macOS
 
 For macOS users, we recommend downloading and installing ddev via [homebrew](https://brew.sh/):
```
Comment (rfay) [inline docs/index.md]: "downloading, installing, and upgrading via"

File: docs/index.md [hunk 3]
```diff
@@ -39,6 +39,9 @@ Linux and macOS end-users can use this line of code to your terminal to download
 curl https://raw.githubusercontent.com/drud/ddev/master/install_ddev.sh | bash
 ```
 
+Later, to upgrade ddev to the newest version, just run the aforementioned `install_ddev.sh` script again.
```
Comment (rfay) [inline docs/index.md]: "just run install_ddev.sh again"

**Comment by tmotyl** [comment]
I've updated the commit, but feel free to take over and finish it.

**Comment by tmotyl** [comment]
great!
Also it would be great to point users to this update instructions, when notifying about new version in the console.
Currently the message looks like
```
A new update is available! please visit https://github.com/drud/ddev/releases/tag/v0.14.0 to download the update. 
```
Which is not very helpful for regular users.
I think we should point users to https://ddev.readthedocs.io/en/latest/#installation
and mention the release page from github as "changelog".

**Comment by rickmanelius** [comment]
I'm good with this. @rfay has been the point person for outside merges, so I'll stay true to that protocol.

**Comment by rfay** [comment]
Thanks so much for this @tmotyl !

**Pull request merged**